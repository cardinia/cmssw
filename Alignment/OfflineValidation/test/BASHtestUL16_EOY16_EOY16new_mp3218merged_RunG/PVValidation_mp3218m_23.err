26-Sep-2019 13:07:39 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/3455B62F-2012-E911-ABFE-0CC47A7452DA.root
26-Sep-2019 13:07:42 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/3455B62F-2012-E911-ABFE-0CC47A7452DA.root
Begin processing the 1st record. Run 279489, Event 170240, LumiSection 1 on stream 0 at 26-Sep-2019 13:07:52.946 CEST
Begin processing the 1001st record. Run 279489, Event 9174389, LumiSection 5 on stream 0 at 26-Sep-2019 13:09:19.677 CEST
Begin processing the 2001st record. Run 279489, Event 17094450, LumiSection 9 on stream 0 at 26-Sep-2019 13:10:33.779 CEST
Begin processing the 3001st record. Run 279489, Event 24115375, LumiSection 13 on stream 0 at 26-Sep-2019 13:11:43.353 CEST
Begin processing the 4001st record. Run 279489, Event 34684051, LumiSection 18 on stream 0 at 26-Sep-2019 13:13:00.091 CEST
Begin processing the 5001st record. Run 279489, Event 42573027, LumiSection 22 on stream 0 at 26-Sep-2019 13:14:19.909 CEST
Begin processing the 6001st record. Run 279489, Event 52070009, LumiSection 26 on stream 0 at 26-Sep-2019 13:15:36.982 CEST
Begin processing the 7001st record. Run 279489, Event 59663591, LumiSection 30 on stream 0 at 26-Sep-2019 13:16:50.745 CEST
Begin processing the 8001st record. Run 279489, Event 69766596, LumiSection 35 on stream 0 at 26-Sep-2019 13:18:11.930 CEST
Begin processing the 9001st record. Run 279489, Event 78416936, LumiSection 39 on stream 0 at 26-Sep-2019 13:19:34.311 CEST
Begin processing the 10001st record. Run 279489, Event 85002618, LumiSection 43 on stream 0 at 26-Sep-2019 13:20:50.027 CEST
Begin processing the 11001st record. Run 279489, Event 95255523, LumiSection 48 on stream 0 at 26-Sep-2019 13:22:11.203 CEST
Begin processing the 12001st record. Run 279489, Event 102869303, LumiSection 52 on stream 0 at 26-Sep-2019 13:23:25.662 CEST
Begin processing the 13001st record. Run 279489, Event 112267233, LumiSection 56 on stream 0 at 26-Sep-2019 13:24:40.651 CEST
Begin processing the 14001st record. Run 279489, Event 120675251, LumiSection 60 on stream 0 at 26-Sep-2019 13:25:48.393 CEST
Begin processing the 15001st record. Run 279489, Event 130638294, LumiSection 65 on stream 0 at 26-Sep-2019 13:26:57.770 CEST
Begin processing the 16001st record. Run 279489, Event 137754543, LumiSection 69 on stream 0 at 26-Sep-2019 13:28:05.602 CEST
Begin processing the 17001st record. Run 279489, Event 146444031, LumiSection 73 on stream 0 at 26-Sep-2019 13:29:10.098 CEST
Begin processing the 18001st record. Run 279489, Event 155502852, LumiSection 78 on stream 0 at 26-Sep-2019 13:30:17.577 CEST
Begin processing the 19001st record. Run 279489, Event 163299829, LumiSection 82 on stream 0 at 26-Sep-2019 13:31:27.709 CEST
Begin processing the 20001st record. Run 279489, Event 173340890, LumiSection 87 on stream 0 at 26-Sep-2019 13:32:39.340 CEST
Begin processing the 21001st record. Run 279489, Event 181531098, LumiSection 91 on stream 0 at 26-Sep-2019 13:33:44.481 CEST
Begin processing the 22001st record. Run 279489, Event 189846217, LumiSection 95 on stream 0 at 26-Sep-2019 13:34:51.091 CEST
Begin processing the 23001st record. Run 279489, Event 198442886, LumiSection 99 on stream 0 at 26-Sep-2019 13:36:00.171 CEST
Begin processing the 24001st record. Run 279489, Event 208397239, LumiSection 104 on stream 0 at 26-Sep-2019 13:37:07.979 CEST
Begin processing the 25001st record. Run 279489, Event 216096605, LumiSection 108 on stream 0 at 26-Sep-2019 13:38:15.046 CEST
Begin processing the 26001st record. Run 279489, Event 224806522, LumiSection 113 on stream 0 at 26-Sep-2019 13:39:34.931 CEST
Begin processing the 27001st record. Run 279489, Event 233438035, LumiSection 117 on stream 0 at 26-Sep-2019 13:40:48.592 CEST
Begin processing the 28001st record. Run 279715, Event 1066196528, LumiSection 638 on stream 0 at 26-Sep-2019 13:41:22.158 CEST
Begin processing the 29001st record. Run 279715, Event 1097622561, LumiSection 655 on stream 0 at 26-Sep-2019 13:41:34.371 CEST
Begin processing the 30001st record. Run 279715, Event 1106598141, LumiSection 660 on stream 0 at 26-Sep-2019 13:41:49.541 CEST
Begin processing the 31001st record. Run 279715, Event 1115365196, LumiSection 664 on stream 0 at 26-Sep-2019 13:42:01.996 CEST
Begin processing the 32001st record. Run 279715, Event 1123443433, LumiSection 668 on stream 0 at 26-Sep-2019 13:42:09.054 CEST
Begin processing the 33001st record. Run 279715, Event 1129839594, LumiSection 672 on stream 0 at 26-Sep-2019 13:42:22.001 CEST
Begin processing the 34001st record. Run 279715, Event 1138612135, LumiSection 676 on stream 0 at 26-Sep-2019 13:42:40.113 CEST
Begin processing the 35001st record. Run 279715, Event 1146294620, LumiSection 681 on stream 0 at 26-Sep-2019 13:42:54.776 CEST
Begin processing the 36001st record. Run 279715, Event 1154438789, LumiSection 686 on stream 0 at 26-Sep-2019 13:43:01.877 CEST
Begin processing the 37001st record. Run 279715, Event 1161787381, LumiSection 690 on stream 0 at 26-Sep-2019 13:43:10.907 CEST
Begin processing the 38001st record. Run 280330, Event 272760759, LumiSection 145 on stream 0 at 26-Sep-2019 13:43:22.299 CEST
Begin processing the 39001st record. Run 280330, Event 280328893, LumiSection 149 on stream 0 at 26-Sep-2019 13:43:31.616 CEST
Begin processing the 40001st record. Run 280330, Event 289820703, LumiSection 154 on stream 0 at 26-Sep-2019 13:43:40.769 CEST
Begin processing the 41001st record. Run 280330, Event 296795730, LumiSection 158 on stream 0 at 26-Sep-2019 13:43:54.224 CEST
Begin processing the 42001st record. Run 280330, Event 308805139, LumiSection 165 on stream 0 at 26-Sep-2019 13:44:06.862 CEST
Begin processing the 43001st record. Run 280330, Event 317226241, LumiSection 169 on stream 0 at 26-Sep-2019 13:44:20.134 CEST
Begin processing the 44001st record. Run 280330, Event 325026764, LumiSection 174 on stream 0 at 26-Sep-2019 13:44:34.059 CEST
Begin processing the 45001st record. Run 280330, Event 332840772, LumiSection 178 on stream 0 at 26-Sep-2019 13:44:48.896 CEST
26-Sep-2019 13:45:00 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/3455B62F-2012-E911-ABFE-0CC47A7452DA.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[8.173MB/8.173MB][100%][==================================================][8.173MB/s]  [8.173MB/8.173MB][100%][==================================================][8.173MB/s]  
