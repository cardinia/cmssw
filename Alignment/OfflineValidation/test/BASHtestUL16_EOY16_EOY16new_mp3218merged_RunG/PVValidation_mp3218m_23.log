000 (3189587.023.000) 09/26 11:25:05 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189587.023.000) 09/26 13:07:23 Started transferring input files
	Transferring to host: <128.142.115.252:9618?addrs=128.142.115.252-9618+[2001-1458-301-90--100-413]-9618&noUDP&sock=13988_8fb2_12541>
...
040 (3189587.023.000) 09/26 13:07:23 Finished transferring input files
...
001 (3189587.023.000) 09/26 13:07:23 Job executing on host: <128.142.115.252:9618?addrs=128.142.115.252-9618+[2001-1458-301-90--100-413]-9618&noUDP&sock=13853_9590_3>
...
006 (3189587.023.000) 09/26 13:07:31 Image size of job updated: 120392
	82  -  MemoryUsage of job (MB)
	83132  -  ResidentSetSize of job (KB)
...
006 (3189587.023.000) 09/26 13:12:31 Image size of job updated: 653056
	532  -  MemoryUsage of job (MB)
	544240  -  ResidentSetSize of job (KB)
...
006 (3189587.023.000) 09/26 13:17:32 Image size of job updated: 653380
	532  -  MemoryUsage of job (MB)
	544556  -  ResidentSetSize of job (KB)
...
006 (3189587.023.000) 09/26 13:22:33 Image size of job updated: 653772
	533  -  MemoryUsage of job (MB)
	544948  -  ResidentSetSize of job (KB)
...
006 (3189587.023.000) 09/26 13:32:33 Image size of job updated: 654932
	534  -  MemoryUsage of job (MB)
	546108  -  ResidentSetSize of job (KB)
...
040 (3189587.023.000) 09/26 13:45:27 Started transferring output files
...
040 (3189587.023.000) 09/26 13:45:27 Finished transferring output files
...
005 (3189587.023.000) 09/26 13:45:27 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:33:16, Sys 0 00:00:14  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:33:16, Sys 0 00:00:14  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	8463  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	8463  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     8416        1   2210582 
	   Memory (MB)          :      534     2000      2000 
...
