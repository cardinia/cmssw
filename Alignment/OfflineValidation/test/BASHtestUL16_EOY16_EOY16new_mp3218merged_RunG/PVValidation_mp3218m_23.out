Job started at  Thu Sep 26 13:07:23 CEST 2019
batch dir: /pool/condor/dir_3682 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_279489_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 8.3M -rw-r--r--. 1 tkello zh 8.2M Sep 26 13:45 PVValidation_mp3218m_279489.root -rw-r--r--. 1 tkello zh 20K Sep 26 13:07 PVValidation_mp3218m_279489_cfg.py -rw-r--r--. 1 tkello zh 7.1K Sep 26 13:45 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 26 13:07 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 13:07 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 13:07 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 13:07 tmp drwx------. 3 tkello zh 4.0K Sep 26 13:07 var
Job ended at  Thu Sep 26 13:45:27 CEST 2019
